lib/ansible/modules/cloud/amazon/ec2_eni.py

```diff
@@ -562,6 +574,7 @@ def main():
             source_dest_check=dict(default=None, type='bool'),
             delete_on_termination=dict(default=None, type='bool'),
             secondary_private_ip_addresses=dict(default=None, type='list'),
+            purge_secondary_private_ip_addresses=dict(default=False, type='bool'),
```

Comment:
I think `secondary_private_ip_addresses` should be required if `purge_secondary_private_ip_addresses` is `True` (because the latter does nothing if the former is not explicitly set)

```
required_if=[['purge_secondary_private_ip_addresses', True, ['secondary_private_ip_addresses']]]
```

Reply:
Addressed.